.github/workflows/snakemake-ci.yml

```diff
@@ -0,0 +1,65 @@
+name: Snakemake CI
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  build-linux:
+    runs-on: ubuntu-latest
+    strategy:
+      max-parallel: 5
+
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up Python 3.10
+        uses: actions/setup-python@v3
+        with:
+          python-version: '3.10'
+
+      - name: Clone repository
+        uses: actions/checkout@v3    
+
+      - name: Install micromamba and create environment
+        uses: mamba-org/setup-micromamba@v1
+        with: 
+          environment-file: environment.yml
```

Comment:
This won't work as the environment file https://github.com/AndriiPovsten/Snakemake-backend-for-RECAST/blob/03ad8f05dd2ae15fa504deb0964b511ab84d6660/recast_helloworld/environment.yml is at `recast_helloworld/environment.yml` and you're just giving `environment.yml`. You need to give a path relative to the top level of the repository. You'll also want to specify the shell as is done in https://github.com/AndriiPovsten/Snakemake-backend-for-RECAST/issues/3#issuecomment-1665957650.